### PR TITLE
fix: fix the missing entry in the index and update the codes

### LINKED ISSUE #1832: Add CRUD docs to the main page
Add `CRUD` docs to the main page

### PR DESCRIPTION
**Pull request by nan-wang**
fix: fix the missing entry in the index and update the codes
resolved https://github.com/jina-ai/jina/issues/1832

### CODE REVIEW

**Comment by nan-wang** [comment]
We might need a better example for the code snippets. Let's do it in another PR.

**Review by cristianmtr — APPROVED** [review]

File: chapters/crud/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The Flow object now supports a `delete` and `update` method, with a signature si
         f.update(input_fn=random_docs(10))
 
     with f:
-        f.delete(input_fn=random_docs(10))
+        f.delete(input_fn=[d.id for d in random_docs(10)])
```
Comment (hanxiao) [inline chapters/crud/index.md]:
```suggestion
        delete_ids = [d.id for d in random_docs(10)]
        f.delete(delete_ids)
```

Comment (hanxiao) [reply chapters/crud/index.md]: writing in this way is more clear that we do not need to construct docs for deleting

**Review by cristianmtr — APPROVED** [review]

**Review by bwanglzu — APPROVED** [review]
LGTM!

**Comment by FionnD** [comment]
@nan-wang can you please merge this if it's all good? :)

**Comment by bwanglzu** [comment]
@nan-wang I'll take over and apply changes ;)

**Pull request merged**